README.md

```diff
@@ -54,7 +54,8 @@ by their priority and status. It can be found [here](https://github.com/orgs/Rev
 Thrive is written in C# with a few helper scripts written in ruby. In
 order to work on the C# you need to compile Thrive yourself. You can
 find instructions for how to do that in the [setup
-instructions][setupguide].
+instructions][setupguide]. And if you've never used Godot before
+please read [learning Godot](LEARNING_GODOT.md).
```

Comment:
This link is wrong, it is missing the folder and it is in the wrong case (so the link doesn't work on Linux).